Rakefile

```diff
@@ -12,7 +12,7 @@ end
 
 task :test => [:check_test]
 
-task :generate_host_config => [:check_acceptance]do |t, args|
+task :generate_host_config do |t, args| #=> [:check_acceptance]do |t, args|
 
   target = ENV["TEST_TARGET"] || 'centos7-64'
   generate = "bundle exec beaker-hostgenerator"
```

Comment:
@er0ck I think we will have to wait for host generator, since it directs the output to a file.  We could do that with a boolean flag but that feels hackish.